examples/tutorials/Part 0 - Installation Guide.md

```diff
@@ -0,0 +1,228 @@
+
+# Part 0: Installing PySyft
+
+Before you start following the tutorials, you need to install PySyft on your computer.
+
+Here are the PySyft installation instructions for Linux, MacOs and Windows.
+
+Just scroll down to find your preferred OS:
+
+--------------
+
+Authors:
+
+- Alan Aboudib - Twitter: [@alan_aboudib](https://twitter.com/alan_aboudib)
+
+
+
+----------------------------
+## 0.1 Linux
+--------------------
+
+Installing PySyft on Linux is really straight forward. Here are the steps:
+
+#### 1. Make sure you have python >= 3.6
+
+You can check out the version number by running:
+
+<font color='red'>$</font>  `python --version`
+
+#### 2. Install pytorch 1.0:
+
+Get the installation command here:
+
+https://pytorch.org/get-started/locally/. (use the pip option)
+
+and run it in the terminal.
+
+**<font color='red'>Attention:</font>** You might need to use `sudo` to run the installation command if you are not using Conda.
+
+
+#### 3. Clone the PySyft repo from github
+
+<font color='red'>$</font> `git clone https://github.com/OpenMined/PySyft.git`
+
+#### 4. Enter the cloned repo
+
+<font color='red'>$</font> `cd PySyft`
+
+#### 5. Install PySyft
+
+<font color='red'>$</font> `sudo python setup.py install`
```

Comment:
this probably should not be done with sudo, additionally, with that in mind, pip install pysyft is almost always the better option as it runs this but also can take advantage of wheel files.

Reply:
Done!
Actually, I had to use `sudo` because I am not inside a virtual env. I could have used `--user` installation option too. So yeah, I will remove this and leave it to the user to deal with their permissions